handbook/company/pricing-features-table.yml

```
@@ -507,7 +507,7 @@
   tier: Premium
 - industryName: Self-managed
   friendlyName: Host it yourself
-  description: Deploy Fleet anywhere and host it yourself, even in airgapped environments.
+  description: Deploy Fleet anywhere and host it yourself.
```

```suggestion
  description: Deploy Fleet anywhere and host it yourself, even in air-gapped environments except where technologically impossible.
```
Hey @dherder, check out the suggestion above, or, we can keep your PR as is and this context to the documentation instead. Why? https://fleetdm.com/handbook/company/why-this-way#why-read-documentation

---

@mikermcneil looks good to me